icicle/include/poseidon/constants/generate_parameters.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python3
+
+# 0.3.7
+import galois
+import numpy as np
+
+import os
+
+# pip install poseidon-hash
+from poseidon import round_constants as rc, round_numbers as rn
+import poseidon
+print(f'poseidon.__file__ = {poseidon.__file__}')
+
+# Modify these
```

Comment:
is this the right location for this file?

Reply:
Yes. But all this directory will be changed down the road.
So, for now I'll leave it as is.